taskcluster/translations_taskgraph/transforms/training_continuation.py

```diff
@@ -0,0 +1,94 @@
+from taskgraph.transforms.base import TransformSequence
+from urllib.parse import urljoin
+import os
+
+CONTINUE_TRAINING_ARTIFACTS = (
+    "devset.out",
+    "model.npz",
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-bleu-detok.npz.decoder.yml",
+    "model.npz.best-ce-mean-words.npz",
+    "model.npz.best-ce-mean-words.npz.decoder.yml",
+    "model.npz.best-chrf.npz",
+    "model.npz.best-chrf.npz.decoder.yml",
+    "model.npz.decoder.yml",
+    "model.npz.optimizer.npz",
+    "model.npz.progress.yml",
+    "model.npz.yml",
+    "train.log",
+    "valid.log",
+    "vocab.spm",
+)
+
+INITIALIZE_MODEL_ARTIFACTS = (
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-ce-mean-words.npz",
+    "model.npz.best-chrf.npz",
+)
+
+
+def get_artifact_mount(url, directory, artifact_name):
+    normalized_url = f"{url}/" if not url.endswith("/") else url
```

Comment:
nitpick: It may be better to run this through `urlparse` rather than try to cleanup the url yourself.

Reply:
Even when I run it through `urlparse` if the input is missing that `/` there are issues using `urljoin` :/